api/V1/Recording.js

```diff
@@ -282,4 +282,102 @@ module.exports = (app, baseUrl) => {
         });
       }
     );
+
+  /**
+  * @api {delete} /api/v1/recordings/:id Delete an existing recording
+  * @apiName DeleteRecording
+  * @apiGroup Recordings
+  *
+  * @apiUse V1UserAuthorizationHeader
+  *
+  * @apiUse V1ResponseSuccess
+  * @apiUse V1ResponseError
+  */
+  app.delete(
+    apiUrl + '/:id',
+    passport.authenticate(['jwt'], { session: false }),
+    async (request, response) => {
+      log.info(request.method + " Request: " + request.url);
+
+      if (!requestUtil.isFromAUser(request))
+        return responseUtil.notFromAUser(response);
+
+      var id = parseInt(request.params.id);
+      if (!id)
+        return responseUtil.invalidDataId(response);
+
+      var deleted = await models.Recording.deleteOne(request.user, id);
+      if (deleted) {
+        responseUtil.send(response, {
+          statusCode: 200,
+          success: true,
+          messages: ["Deleted recording."],
+        });
+      } else {
+        responseUtil.send(response, {
+          statusCode: 400,
+          success: false,
+          messages: ["Failed to delete recording."],
+        });
+      }
+    }
+  );
+
+  /**
+  * @api {patch} /api/v1/recordings/:id Update an existing recording
+  * @apiName UpdateRecording
+  * @apiGroup Recordings
+  * @apiDescription This call is used for updating fields of current recordings.
+  * The only values that can be updated at the moment are the following ones.
+  * If any other values are requested to be updated the request will fail and no
+  * updates will occur.
+  * - location
+  * - comment
```

Comment:
Something like this might be better:

The following fields may be updated are:
- location
- comment

If a change to any other field is attempted the request will fail and no update will occur.